pyproject.toml

```diff
@@ -10,6 +10,7 @@ dependencies = [
     "asgiref>=3.8.1",
     "sqlparse>=0.3.1",
     "tzdata; sys_platform == 'win32'",
+    "tomli; python_version < '3.11'",
```

Comment:
If we really wanted to avoid this dependency, we could only implement this feature if `tomllib` is available in the stdlib. But I think that would be a worse experience for users than just including `tomli` on older supported Python versions.

Reply:
nessita said (not mentioning to avoid spam)

> For future reference, it's very likely that we won't accept adding a new dependency. Historically Django has worked hard to keep the dependency requirements to an absolute minimum so this work should plan around that.

Reply:
Another alternative might be to vendor the library. I tend to think the distaste for dependencies was initiated by a time before dependency management was a think in Python, and that a backport of a built-in module is pretty obvious candidate for an exception, but vendoring the backport or limiting a feature to Python 3.11+ might be acceptable alternatives.